### PR TITLE
Introduce manual post splitting

### PR DESCRIPTION
**Pull request by wm75**
... and count links on mastodon as fixed 23 chars

With this users can specify manual splits into sub-posts by including triple newlines in the content for plugins that support it.

### CODE REVIEW

File: lib/plugins/bluesky.py [hunk 1]
```diff
@@ -158,15 +158,21 @@ def handle_url_card(
             )
         return embed_external
 
+    def content_in_chunks(content, max_chunk_length):
```
Comment (arash77) [inline lib/plugins/bluesky.py:161]:
```suggestion
    def content_in_chunks(self, content, max_chunk_length):
```

File: lib/plugins/mastodon.py [hunk 2]
```diff
@@ -14,21 +14,28 @@ def __init__(self, **kwargs):
         )
         self.max_content_length = kwargs.get("max_content_length", 500)
 
+    def content_in_chunks(content, max_chunk_length):
```
Comment (arash77) [inline lib/plugins/mastodon.py:17]:
```suggestion
    def content_in_chunks(self, content, max_chunk_length):
```

File: lib/plugins/slack.py [hunk 3]
```diff
@@ -11,15 +11,21 @@ def __init__(self, **kwargs):
         self.channel_id = kwargs.get("channel_id")
         self.max_content_length = kwargs.get("max_content_length", 40000)
 
+    def content_in_chunks(content, max_chunk_length):
```
Comment (arash77) [inline lib/plugins/slack.py:14]:
```suggestion
    def content_in_chunks(self, content, max_chunk_length):
```

**Review by arash77 — COMMENTED** [review]
We need to address the following error: `content_in_chunks() takes 2 positional arguments but 3 were given`.

**Comment by wm75** [comment]
lol, me copy-pasting code around without properly testing the result :roll_eyes: 
Sorry!

**Comment by arash77** [comment]
No worries! 😅

**Review by arash77 — APPROVED** [review]

**Pull request merged**